### PR TITLE
ASoC: codecs: add support for ES8326 dts

### PR DESCRIPTION
**Pull request by yangxiaohua2009**
The ES8326 codec is not compatible with ES8316 and requires a dedicated driver.

This is a separate patch for ES8326 dts

Codec patch in #3508 

Signed-off-by: David Yang <yangxiaohua@everest-semi.com>

### CODE REVIEW

**Comment by sofci** [comment]
Can one of the admins verify this patch?
> reply **test&nbsp;this&nbsp;please** to run this test once

File: Documentation/devicetree/bindings/sound/everest,es8326.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
```
Comment (plbossart) [inline Documentation/devicetree/bindings/sound/everest,es8326.yaml:1]: typo in commit message: separate.

File: Documentation/devicetree/bindings/sound/everest,es8326.yaml [hunk 2]
```diff
@@ -0,0 +1,110 @@
+# SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/sound/everest,es8326.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
+
+title: Everest ES8326 audio CODEC
+
+maintainers:
+  - Daniel Drake <drake@endlessm.com>
```
Comment (plbossart) [inline Documentation/devicetree/bindings/sound/everest,es8326.yaml]: I don't think Daniel Drake is a maintainer for this codec.

**Review by plbossart — CHANGES_REQUESTED** [review]

**Comment by plbossart** [comment]
closing, this will be handled with an upstream merge

**Pull request closed without merging**